tests/e2e/slashing.go

```diff
@@ -63,15 +63,14 @@ func (s *CCVTestSuite) TestRelayAndApplySlashPacket() {
 		s.Require().Nil(err)
 		consAddr := sdk.GetConsAddress(pubkey)
 		// map consumer consensus address to provider consensus address
-		if providerAddr, found := providerKeeper.GetValidatorByConsumerAddr(
+		providerAddr, found := providerKeeper.GetValidatorByConsumerAddr(
 			s.providerCtx(),
 			s.consumerChain.ChainID,
 			consAddr,
-		); found {
-			consAddr = providerAddr
```

Comment:
It's notable that setting `consAddr = providerAddr` was concealing this bug previously, since `SetOutstandingDowntime` was being called on the consumer keeper with the provider address. We should learn from this and use type safety more often! See #707